core/eth/client.go

```diff
@@ -128,12 +128,14 @@ func (client *CallerSubscriberClient) GetTxReceipt(hash common.Hash) (*TxReceipt
 	return &receipt, err
 }
 
+// GetBlockHeight returns the block height.
```

Comment:
This boilerplate is worse than unhelpful. If we can't have decent comments, let's turn the check off.